test.sh

```diff
@@ -63,7 +63,8 @@ else
   run go test ${dirlist}
 fi
 
-echo "Checking for unformatted files:"
+setsid bash test/integration-test.sh
```

Comment:
In the absence of setsid, if you run test.sh multiple times, it will leave a dangling copy of boulder running, which will mean subsequent runs won't pick up boulder changes. I'm going to poke at an install/run command pair which will fix these issues with killing grandchildren (since there will be no grandchildren).